### ISSUE TITLE
@nativescript/core update  to 7.1.2 breaks aot/uglify with fresh Angular app: Cannot read property '_executingContext' of undefined

### ISSUE THREAD

**Issue by james-criscuolo** [NativeScript/NativeScript#9166]
**Environment**
Provide version numbers for the following components (information can be retrieved by running `tns info` in your project folder or by inspecting the `package.json` of the project):
 - CLI: 7.1.2 (latest)
 - Cross-platform modules: none
 - Android Runtime: 7.0.1
 - iOS Runtime: N/A
 - XCode Version: N/A
 - Plugin(s): N/A

**Describe the bug**
Using flags `--env.aot --env.uglify` does not work when updating to core 7.1.2 from 7.1.1 (no other dependency changes) in an angular project. I assume this affects ios as well, since the [code I suspect](https://github.com/NativeScript/NativeScript/commit/7a79a8988b55e7e0896e7ad38f571aba6987edb5) is for both platforms. That said, the stack trace is android-specific, so I'm not sure, I can investigate further if necessary.

**To Reproduce**

```
ns create test-app
> select angular
> select hello world
cd test-app
ns prepare android --compileSdk 29 && ns build android --compileSdk 29 && ns run android --compileSdk 29 --env.aot --env.uglify
```
**Expected behavior**
The app starts. Instead here is the stack trace:

```
System.err: Calling js method onAnimationEnd failed
System.err: TypeError: Cannot read property '_executingContext' of undefined
System.err:
System.err: StackTrace:
System.err: R(file: node_modules/@nativescript/core/ui/frame/fragment.transitions.android.js:537:40)
System.err: 	at n.onAnimationEnd(file: node_modules/@nativescript/core/ui/frame/fragment.transitions.android.js:177:8)
System.err: 	at com.tns.Runtime.callJSMethodNative(Native Method)
System.err: 	at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1302)
System.err: 	at com.tns.Runtime.callJSMethodImpl(Runtime.java:1188)
System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:1175)
System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:1153)
System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:1149)
System.err: 	at java.lang.Object_vendor_22932_22_n.onAnimationEnd(Unknown Source:10)
System.err: 	at android.animation.Animator$AnimatorListener.onAnimationEnd(Animator.java:554)
System.err: 	at android.animation.AnimatorSet.endAnimation(AnimatorSet.java:1301)
System.err: 	at android.animation.AnimatorSet.doAnimationFrame(AnimatorSet.java:1086)
System.err: 	at android.animation.AnimationHandler.doAnimationFrame(AnimationHandler.java:146)
System.err: 	at android.animation.AnimationHandler.access$100(AnimationHandler.java:37)
System.err: 	at android.animation.AnimationHandler$1.doFrame(AnimationHandler.java:54)
System.err: 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:964)
System.err: 	at android.view.Choreographer.doCallbacks(Choreographer.java:790)
System.err: 	at android.view.Choreographer.doFrame(Choreographer.java:721)
System.err: 	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:951)
System.err: 	at android.os.Handler.handleCallback(Handler.java:883)
System.err: 	at android.os.Handler.dispatchMessage(Handler.java:100)
System.err: 	at android.os.Looper.loop(Looper.java:214)
System.err: 	at android.app.ActivityThread.main(ActivityThread.java:7356)
System.err: 	at java.lang.reflect.Method.invoke(Native Method)
System.err: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
System.err: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
```

**Sample project**
See reproduction steps, an `ns create` app is all that is needed.

**Comment by jessorlisa** [comment]
https://github.com/NativeScript/nativescript-cli/issues/5481#issuecomment-765259091

**Comment by james-criscuolo** [comment]
Confirmed this is now fixed with today's releases (7.2.0/4.1.0), closing. Thanks!
